assets/queries/ansible/aws/public_ecr_policy/test/positive.yaml

```diff
@@ -0,0 +1,13 @@
+- name: set-policy as object
+  community.aws.ecs_ecr:
+    name: needs-policy-object
+    policy:
+      Version: '2008-10-17'
+      Statement:
+        - Sid: read-only
+          Effect: Allow
+          Principal: '*'
+          Action:
+            - ecr:GetDownloadUrlForLayer
+            - ecr:BatchGetImage
```

Comment:
This policy statements should be inside a json specification, because this is cloudformation syntax

Reply:
This is an example from the ansible documentation, I think that this way is supported and I realized I should analyze the policy as a string/JSON, already fixed.

https://docs.ansible.com/ansible/latest/collections/community/aws/ecs_ecr_module.html#parameter-policy